geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/data/HBaseConnectionPool.scala

```diff
@@ -101,6 +101,7 @@ object HBaseConnectionPool extends LazyLogging {
       ProvidedConnection(ConnectionParam.lookup(params))
     } else {
       val conf = getConfiguration(params)
+      logger.debug(s"Connecting to HBase instance at ${conf.get(HConstants.ZOOKEEPER_QUORUM)}")
```

Comment:
I wonder if we should add logic to add warning if the ZooKeeper is localhost.  

Meh, probably too much.

Reply:
we do have a warning about that already actually.